code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -2781,15 +2781,15 @@
 	flavor = newflavor
 
 	switch(newflavor)
-		if("boneless pork ribs", "grilled chicken", "pizza square", "spaghetti chunks", "chicken tender")
+		if("boneless pork ribs", "grilled chicken", "pizza square", "spaghetti", "red crayon")
```

Comment:
please do not try to sneak in a chicken tendie removal in your PRs again or I will have to gitban you

Reply:
i take my chicken tendies very serious